src/components/views/messages/ReactionsRowButton.tsx

```diff
@@ -105,22 +108,56 @@ export default class ReactionsRowButton extends React.PureComponent<IProps, ISta
         }
 
         const room = this.context.getRoom(mxEvent.getRoomId());
+        const customReactionImagesEnabled = SettingsStore.getValue("feature_render_reaction_images");
```

Comment:
Might be good to hoist this to a common parent and share via props/context, each event having to do this can add up to a lot of localStorage accesses which can cause performance issues